src/quantuminspire/projectq/backend_qx.py

```diff
@@ -63,8 +64,12 @@ def __init__(self, num_runs: int = 1024, verbose: int = 0, quantum_inspire_api:
         self._measured_ids: List[int] = []
         self._allocated_qubits: Set[int] = set()
         self._max_qubit_id: int = -1
-        if not quantum_inspire_api:
-            raise RuntimeError("Api is required")
+        if quantum_inspire_api is None:
+            try:
+                quantum_inspire_api = QuantumInspireAPI()
+            except ApiError as ex:
+                raise ApiError('Make sure you have saved your token credentials on disk (see README.md) or '
```

Comment:
If you `pip install` this from the cheeseshop, will you be able to find README.md ?

Reply:
Nope, I'll remove that reference to README.md. I assume that someone who is going to use quantuminspire SDK can find this information op pypi (or github).